src/resources/FileUpload.js

```diff
@@ -16,10 +17,17 @@ const sign = async (req, res) => {
 
     const s3 = new AWS.S3({ signatureVersion: "v4" });
 
+    let bucketName, region;
+    try {
+      bucketName = FormioConfig.settings.fileUpload.aws.bucket;
+      region = FormioConfig.settings.fileUpload.aws.region;
+    } catch(error) {
```

Comment:
Adding try catch for handelling when the config is not updated in BK seems a bit much. Generally QAs look  at the ticket if anything needs to be updated in BK and make the necessary changes in config before deployment. I think it can be assumed that the config is updated when the code is ran.

Reply:
Removed try-catch block